pages/tools/components/ToolingTable.tsx

```diff
@@ -189,9 +189,9 @@ const ToolingTable = ({
                     >
                       <TableCell
                         attributes={{
-                          className: `${tool.name.split(' ').some((segment) => segment.length > 25) ? 'break-all' : ''} gap-x-2 gap-y-1`,
+                          className: `${tool.name.split(' ').some((segment) => segment.length > 25) ? 'break-word' : ''} gap-x-2 gap-y-1`,
```

Comment:
note: Nevermind, I saw now why you need it. is because some packages don't have any space or `-`, like "LateApexEarlySpeed.Json.Schema". Closing this then